src/main/java/com/atlassian/jira/cloud/jenkins/util/XmlUtils.java

```diff
@@ -0,0 +1,42 @@
+package com.atlassian.jira.cloud.jenkins.util;
+
+import hudson.XmlFile;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.io.StringReader;
+
+public class XmlUtils {
```

Comment:
Same here: some Javadoc comments will help future devs easily identify the purpose of this class.